src/Serializer/Normalizer/EmptyObjectNormalizer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\FanCourierBundle\Serializer\Normalizer;
+
+use Answear\FanCourierBundle\Request\RequestInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+class EmptyObjectNormalizer implements NormalizerInterface
+{
+    public function getSupportedTypes(?string $format): array
+    {
+        return [
+            RequestInterface::class => true,
+        ];
+    }
+
+    public function normalize(mixed $object, ?string $format = null, array $context = []): array
+    {
+        return [];
+    }
+
+    public function supportsNormalization($data, $format = null): bool
+    {
+        return is_object($data) && $this->isEmptyClass($data);
```

Comment:
Is this normalizer necessary?
Why not `https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Serializer/Normalizer/CustomNormalizer.php

Reply:
I checked this, and indeed no normalizer is needed for normalizing requests.